tmva/sofie/inc/TMVA/SOFIE_common.hxx

```diff
@@ -145,7 +149,8 @@ public:
    }
 
 private:
-   bool        fConstant = false;   ///< Flag specifying if tensor is a Constant one (coming from a Constant operator)
+   bool  fConstant = false;      ///< Flag specifying if tensor is a Constant one (coming from a Constant operator)
```

Comment:
will it be better to use enums here to represent the not-writable property?

Reply:
Yes, this could be an enum, let's maybe add this when we introduce a new property